doc/source/rest.j2

```diff
@@ -601,30 +601,48 @@ Gnocchi's search API supports to the ability to execute a query across
 |resources| or |metrics|. This API provides a language to construct more
 complex matching contraints beyond basic filtering.
 
-Usage
------
+Usage and format
+----------------
 
 You can specify a time range to look for by specifying the `start` and/or
 `stop` query parameter, and the |aggregation method| to use by specifying the
 `aggregation` query parameter.
 
+Query can be expressed into two formats `JSON` or a `STRING`.
+
 The supported operators are: equal to (`=`, `==` or `eq`), lesser than (`<` or
 `lt`), greater than (`>` or `gt`), less than or equal to (`<=`, `le` or `≤`)
 greater than or equal to (`>=`, `ge` or `≥`) not equal to (`!=`, `ne` or `≠`),
 addition (`+` or `add`), substraction (`-` or `sub`), multiplication (`*`,
-`mul` or `×`), division (`/`, `div` or `÷`). These operations take either one
-argument, and in this case the second argument passed is the value, or it.
+`mul` or `×`), division (`/`, `div` or `÷`). In JSON format, these operations
+take either one argument, and in this case the second argument passed is the
+value, or it. In STRING format, this is just `<first argument> <operator>
```

Comment:
In JSON format, these operations take only one argument, the second argument being automatically set to the field value.

(or the measure value, not sure what it applies to in this paragraph)